app/assets/stylesheets/common/foundation/variables.scss

```diff
@@ -75,6 +75,7 @@ $line-height-large: 1.4; // Normal or small text
 // These files don't actually exist. They're injected by Stylesheet::Compiler.
 // --------------------------------------------------
 
+@import "fonts";
```

Comment:
The variables scss file is included in each stylesheet so importing the fonts here will inject the font styles to each stylesheet too (plugins, themes, components, mobile, desktop). 

I think you can load the selected @font-face styles in `foundation/base.scss` and the full list of styles and classes in in `wizard.scss`.

Reply:
I added `@import "font"` to `common.scss`, as I used to do before switching to the "special import".